kind/actions/cluster_actions.go

```diff
@@ -149,13 +150,23 @@ func KubeadmInit(clusterName string) error {
 	if err != nil {
 		return err
 	}
+	// Upload certs flag changed to non-experimental in >= 1.15
+	uploadCertsFlag := "--experimental-upload-certs"
+	parsedVersion, err := k8sverison.ParseGeneric(version)
```

Comment:
should be ParseSemantic? generic will fail parsing a k8s version with pre-release + metadata.

Reply:
that's ok, we won't be using those tags (yet). if we do end up needing it we can change it later